config/environments/development.rb

```diff
@@ -49,7 +49,7 @@
 
   config.action_mailer.default_url_options = { :host => 'localhost', protocol: 'https' }
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.perform_deliveries = false
+  config.action_mailer.perform_deliveries = true
```

Comment:
did you mean to leave this turned on?  We briefly chatted about installing something that would forward all dev. emails to the coder, but wanted to confirm that's happening here

Reply:
That couldn't happen - we're connected to different schemas.  However, if you boot your local instance and connect to production, you could do that... this was mainly so I could test this feature, then I realized that the whole reason I turned it off before (quota impact with Sendgrid) doesn't apply anymore.

However, point of accidental misuse taken.  I'll turn this off until we implement the feature you talked about (it's easy enough to turn on/off locally).